automation/src/test/scala/org/broadinstitute/dsde/test/api/AuthDomainGroupRoleSpec.scala

```diff
@@ -0,0 +1,220 @@
+package org.broadinstitute.dsde.test.api
+
+import org.broadinstitute.dsde.test.util.AuthDomainMatcher
+import org.broadinstitute.dsde.workbench.auth.AuthToken
+import org.broadinstitute.dsde.workbench.config.UserPool
+import org.broadinstitute.dsde.workbench.fixture.{BillingFixtures, GroupFixtures, WorkspaceFixtures}
+import org.broadinstitute.dsde.workbench.service.BillingProject.BillingProjectRole
+import org.broadinstitute.dsde.workbench.service.Orchestration.groups.GroupRole
+import org.broadinstitute.dsde.workbench.service.{Rawls, RestException, Sam}
+import org.scalatest.{FreeSpec, Matchers}
+
+import scala.util.Try
+
+class AuthDomainGroupRoleSpec extends FreeSpec with WorkspaceFixtures with BillingFixtures with GroupFixtures with Matchers {
+
+  "removing permissions from workspace with auth domain" - {
+
+    "+ project owner, + group member, create workspace, - group member" in {
```

Comment:
Can we change this test so that we are using the `billingProjectOwner`'s token to create the auth domain group and the workspace and then we are using the `student`'s token to check the accessibility and visibility of the workspace? Basically switching `billingProjectOwner` and `student` in this test to be consistent with the other tests in this suite?

Reply:
👍